test/e2e/test_404.py

```diff
@@ -0,0 +1,29 @@
+from pathlib import Path
+from pytest import fixture
+from playwright.sync_api import Page, expect
+
+
+"""
+Code here will run for every test in this file
+"""
+@fixture(autouse=True)
+def setup(page: Page):
+    page.goto("/404.html")
```

Comment:
I say test a random (optionally clever or humorous) URL of your choosing!

Reply:
You have to use the 404 page when you run the test locally. If you use a random page with the local development environment, the 404 page will not load. If we are fine with that test failing locally every time, then I’ll make something random.

Reply:
that's very odd...I'll have to dig into this more, because when I run the `hugo serve -D` locally (outside of the container with the snap'ed version of hugo) it worked perfectly fine... :thinking: 

![image](https://user-images.githubusercontent.com/10230166/200125940-746b3157-58d6-49ec-ba84-0bfcf500cf21.png)

Reply:
Ah....so it looks like it was an improvement between 101 -> 105 when the `hugo serve` command received 404 support.

https://github.com/gohugoio/hugo/releases/tag/v0.103.0

Which is why it was working for me, because hugo just updates in the background with the snap package. :sweat_smile:

Reply:
That would do it! So I'm guessing we should plan on migrating to 105 then?

Reply:
Once that PR is merged, I will update to a random url (maybe randomly generated? Or give me some ideas for fun urls!). In the meantime I will keep it as 404.html